docs/guide/deployer.md

```diff
@@ -46,6 +46,32 @@ await deployer.deployASA("ASA-2", {...}, { reserve: multsigaddr }); // this will
 For opting in to ASA, `deployer` supports two methods :-
 - `optInAcountToASA` to opt-in to a single account signed by secret key of sender.
 - `optInLsigToASA` to opt-in to a contract account (say escrow) where the account is represented by the logic signature address (`lsig.address()`).
+    - To opt in to ASA you can use either `Asset Index` or `name of the ASA`.
+    - Using Asset Index is useful when asset is not deployed using deployer.
+
+- There is one more method which you can use to opt-in, It can be used with group transactions also
+- `executeTransaction` to opt-in single account or contract account to ASA.
+- Ex: To opt-in a single account, Params will look like this:
+```js
+   const execParam: ExecParams = {
+    type: TransactionType.OptInASA,
+    sign: SignType.SecretKey,
+    fromAccount: user.account,
+    assetID: assetID,
+    payFlags: payFlags
+  };
+```
+- Ex: To opt-in to a contract account
+```js
+   const execParam: ExecParams = {
+    type: TransactionType.OptInASA,
+    sign: SignType.LogicSignature,
+    fromAccountAddr: lsig.address(),
+    assetID: assetID,
+    payFlags: payFlags
+    lsig: lsig
+  };
+```
```

Comment:
all things below "There is one more..." should be in the same bulletpoint.

Reply:
done